docs/spec/resource-scopes.md

```diff
@@ -144,4 +144,5 @@ module myMod './path/to/module.bicep' = {
 }
 ```
 
-[arm-scopes]: https://docs.microsoft.com/azure/azure-resource-manager/management/overview#understand-scope
\ No newline at end of file
+⮴📖 [resourceGroup properties] https://docs.microsoft.com/en-us/azure/templates/microsoft.resources/resourcegroups?tabs=bicep
```

Comment:
remove `/en-us` from any docs.microsoft.com links. This prevents the browser from setting the locale correctly.